site-src/geps/gep-1323.md

```diff
@@ -0,0 +1,177 @@
+# GEP 1323: Response Header Filter
+* Issue: [#1323](https://github.com/kubernetes-sigs/gateway-api/issues/1323)
+* Status: Provisional
```

Comment:
```suggestion
* Status: Implementable
```

Unless this winds up being unexpectedly controversial, I think we can move directly to Implementable.

Reply:
The only thing that _might_ be controversial that I can see is that I think it's technically up for debate whether the functionality be core or extended? _maybe_? but ultimately I think you're right.